scripts/client-sdk-unit-tests.sh

```diff
@@ -20,29 +20,16 @@ make rosetta_lib_encodings
 echo "Running"
 ./_build/default/src/lib/rosetta_lib/test/test_encodings.exe > encodings.consensus
 
-# native/nonconsensus
```

Comment:
Cc @psteckler . Unfortunately, these tests broke with the fix, but we didn’t have time to investigate further. I think it’s okay for now, we can open an issue to fix

Reply:
Added #8969.